content/en/getting_started/integrations/azure.md

```diff
@@ -1,180 +1,366 @@
 </div>
 
 ## Setup
 
-{{% site-region region="us,us5,eu,ap1,gov,ap2" %}}
+Follow the instructions on this page to set up the **Azure integration** through an app registration, available for all Datadog sites. 
+
+{{< img src="/getting_started/integrations/azure/GSwAzure_siteSelector.mp4" alt="Site selector for US3 site" video=true >}}
 
-Follow the instructions on this page to set up the **Standard Azure integration**, which is available for all Datadog sites. 
+{{% collapse-content title="Quickstart (recommended)" level="h4" expanded=false id="azure-quickstart-setup" %}}
 
-**Note**: If you are on the US3 site, you can use the **Azure Native integration** for enhanced functionality and streamlined setup. Change the site selector on the right side of this page for the instructions on how to set this up.
+### Choose this if...
 
-{{< img src="/getting_started/integrations/azure/GSwAzure_siteSelector.mp4" alt="Site selector for US3 site" video=true >}}
+- You are setting up Datadog for the first time.
+- You prefer a UI-based workflow and want to minimize the time it takes to create a service principal with the required monitoring permissions.
+- You want to automate setup steps in scripts or CI/CD pipelines.
+
+### Instructions
+
+1. In the Azure integration tile, click **+ Add New App registration**, then select **Quickstart**.
+2. Copy the setup script, and run it in the Azure Cloud shell.
+3. Return to the Datadog UI. You should see **CONNECTED** at the top right corner of the setup script.
+4. Select the subscriptions and management groups to collect data from.
+5. Optionally, click the metric collection toggle to disable all metric collection from Azure. You can also expand the **Advanced Configuration** dropdown to filter metrics by:
+   - Resource provider
+   - Tags
+   - Hosts
+   - App Service Plans
+   - Container Apps
+
+You can also click to enable custom metric collection from [Azure Application Insights][36], and disable the collection of usage metrics.
+
+6. Optionally, click the resource collection toggle to disable the collection of configuration information from your Azure resources.
+7. Enable log collection to setup and configure the services and diagnostic settings needed to forward logs to Datadog:
+   1. If a log forwarder already exists in the tenant, extend its scope to include any new subscriptions or management groups.
+   2. If you're creating a new log forwarder:
+      1. Enter a resource group name to store the log forwarder control plane
+      2. Select a control plane subscription for the log-forwarding orchestration (LFO).
+      3. Select a region for the control plane.<br>
+   **Note**: This section only appears when creating a new log forwarder.
```

Comment:
Nit, if I didn't know better it would be a little unclear to me if this referred to 7.b or just 7.b.iii

Reply:
Clarified in [349564f](https://github.com/DataDog/documentation/pull/32186/commits/349564fa2a078bd75c9195001e8b1e6b628fc23a)